designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview-field-recon.component.html

```diff
@@ -283,7 +283,7 @@ <h3 class="text-center">
                             </div>
                             <div class="entity-meta-field">
                                 <div class="entity-meta-label-multi" data-ng-if="mission.value.authors.length">
-                                    Author(s)
+                                    Author(s) - DEBUG
```

Comment:
I think you left this in on accident. It looks good so far. I'm going to pull and test it soon.

Reply:
Oops!